src/photosui.cs

```diff
@@ -227,6 +247,29 @@ interface PHProjectTypeDescription : NSSecureCoding {
 
 		[Export ("initWithProjectType:title:description:image:")]
 		IntPtr Constructor (NSString projectType, string localizedTitle, [NullAllowed] string localizedDescription, [NullAllowed] UIImage image);
+
+		[Mac (10,14, onlyOn64: true)]
+		[Export ("initWithProjectType:title:attributedDescription:image:subtypeDescriptions:")]
+		[DesignatedInitializer]
+		IntPtr Constructor (NSString projectType, string localizedTitle, [NullAllowed] NSAttributedString localizedAttributedDescription, [NullAllowed] UIImage image, PHProjectTypeDescription[] subtypeDescriptions);
```

Comment:
Do we know what `NSString projectType` is? Is it a smart enum? can we make a better API here? if so you might be able to use BindAs